where.ts

```diff
@@ -0,0 +1,14 @@
+
+import curryN from "./utils/curry_n.ts"
+import { Curry2 } from "./utils/types.ts"
+
+function where(specs : any , testObj: any) {
+  for (let property in specs) {
+    if (Object.prototype.hasOwnProperty.call(specs, property) && !specs[property](testObj[property])) {
```

Comment:
since if condition is long, break in multiple lines... see `whereAll.ts` line 18